src/spac/data_utils.py

```diff
@@ -55,6 +55,11 @@ def ingest_cells(dataframe,
         all_columns
     )
 
+    if len(all_features) == 0:
+        error_message = "Provided regex patern(s) or feature(s) does " + \

```

Comment:
Thanks @ruiheesi, let's add in the error message the regex patterns (one per line between quotes) and the features names (one per line between quotes) for more context to the user. This way users can see any potential extra spaces they skipped.